src/holofuel/pages/CreateOfferRequest/CreateOfferRequest.js

```diff
@@ -25,7 +25,10 @@ const AGENT_ID_LENGTH = 63
 const FormValidationSchema = yup.object().shape({
   counterpartyId: yup.string()
     .required()
-    .length(AGENT_ID_LENGTH),
+    // the first four chars === 'HcSc' (ie: the hc prefix)
+    // .substring(0, 5).matches(/(HcSc)/)
```

Comment:
let's remove this commented code.

Reply:
I do think we should implement a prefix check, but I'll remove the comment here and just make a PT chore for this. :+1: